docs/data-sources/organization_project.md

```diff
@@ -35,8 +35,9 @@ data "aiven_organization_project" "example" {
 ### Read-Only
 
 - `billing_group_id` (String) Billing group ID to assign to the project.
+- `ca_cert` (String, Sensitive) The CA certificate for the project. This is required for configuring clients that connect to certain services like Kafka.
 - `id` (String) The ID of this resource.
-- `parent_id` (String) Link a project to an [organization or organizational unit](https://aiven.io/docs/platform/concepts/orgs-units-projects) by using its ID. To set up proper dependencies please refer to this variable as a reference.
+- `parent_id` (String) Link a project to an [organization or organizational unit](https://aiven.io/docs/platform/concepts/orgs-units-projects) by using its account ID. To set up proper dependencies please refer to this variable as a reference.
```

Comment:
`parent_id` should accept both account_id `a12345` or organization_id `org56789` to simply user experience. Nevertheless TF should always translate `org56789` to `a12345`. We did the same for the old project resource.

Reply:
This is whats happening. Do you want to remove the word `account` from the description?

Reply:
Done